.gitignore

```diff
@@ -35,6 +38,9 @@ hs_err_pid*
 
 *.iml
 
+#
+# idee
+.idee
```

Comment:
idee?

Reply:
It's where my custom IDE is storing project specific configuration.